tiledb/sm/subarray/subarray.cc

```diff
@@ -1608,7 +1608,14 @@ Status Subarray::compute_est_result_size(ThreadPool* const compute_tp) {
       }
 
       if (array_schema->is_nullable(names[i])) {
-        min_size_validity = cell_num * constants::cell_validity_size;
```

Comment:
Why isn't this correct?

Reply:
Fixed --

The estimated validity values now matches the number of cells for both the above scenarios. I'll make a pass over the core to ensure the data model matches the actual usage, but this API can be checked in independently.